### PR TITLE
Fix minor issues with WebAuthn on Factors API doc

### PR DESCRIPTION
**Pull request by mpeng-okta**
## Description:
- Remove stateToken references for WebAuthn on Factors API doc
- Add closing quotation mark
- This PR is not related to a Monolith release -- it is correcting the doc to be accurate to existing monolith

### Resolves:

* [OKTA-329374](https://oktainc.atlassian.net/browse/OKTA-329374)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 1]
```diff
@@ -2614,9 +2613,8 @@ curl -v -X POST \
 -H "Content-Type: application/json" \
 -H "Authorization: SSWS ${api_token}" \
 -d '{
-  "attestation: "o2NmbXRmcGFja2VkZ2F0dFN0bXSiY2FsZyZjc2lnWEgwRgIhAMvf2+dzXlHZN1um38Y8aFzrKvX0k5dt/hnDu9lahbR4AiEAuwtMg3IoaElWMp00QrP/+3Po/6LwXfmYQVfsnsQ+da1oYXV0aERhdGFYxkgb9OHGifjS2dG03qLRqvXrDIRyfGAuc+GzF1z20/eVRV2wvl6tzgACNbzGCmSLCyXx8FUDAEIBvWNHOcE3QDUkDP/HB1kRbrIOoZ1dR874ZaGbMuvaSVHVWN2kfNiO4D+HlAzUEFaqlNi5FPqKw+mF8f0XwdpEBlClAQIDJiABIVgg0a6oo3W0JdYPu6+eBrbr0WyB3uJLI3ODVgDfQnpgafgiWCB4fFo/5iiVrFhB8pNH2tbBtKewyAHuDkRolcCnVaCcmQ==",
```
Comment (mpeng-okta) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md:2617]: Add closing `"` for `"attestation"`

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 2]
```diff
@@ -2582,7 +2582,6 @@ Activate a WebAuthn Factor by verifying the attestation and client data.
 | attestation       | Base64-encoded attestation from the WebAuthn JavaScript call        | Body       | String   | TRUE     |
 | clientData        | Base64-encoded client data from the WebAuthn JavaScript call        | Body       | String   | TRUE     |
 | factorId          | `id` of the Factor returned from enrollment                         | URL        | String   | TRUE     |
-| stateToken        | [State token](#state-token) for the current transaction             | Body       | String   | TRUE     |
```
Comment (mpeng-okta) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md:2585]: This is inaccurate -- there is no state token used here, and it was mistakenly added originally.

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 3]
```diff
@@ -2614,9 +2613,8 @@ curl -v -X POST \
 -H "Content-Type: application/json" \
 -H "Authorization: SSWS ${api_token}" \
 -d '{
-  "attestation: "o2NmbXRmcGFja2VkZ2F0dFN0bXSiY2FsZyZjc2lnWEgwRgIhAMvf2+dzXlHZN1um38Y8aFzrKvX0k5dt/hnDu9lahbR4AiEAuwtMg3IoaElWMp00QrP/+3Po/6LwXfmYQVfsnsQ+da1oYXV0aERhdGFYxkgb9OHGifjS2dG03qLRqvXrDIRyfGAuc+GzF1z20/eVRV2wvl6tzgACNbzGCmSLCyXx8FUDAEIBvWNHOcE3QDUkDP/HB1kRbrIOoZ1dR874ZaGbMuvaSVHVWN2kfNiO4D+HlAzUEFaqlNi5FPqKw+mF8f0XwdpEBlClAQIDJiABIVgg0a6oo3W0JdYPu6+eBrbr0WyB3uJLI3ODVgDfQnpgafgiWCB4fFo/5iiVrFhB8pNH2tbBtKewyAHuDkRolcCnVaCcmQ==",
-  "clientData": "eyJjaGFsbGVuZ2UiOiJVSk5wYW9sVWt0dF9vcEZPNXJMYyIsIm9yaWdpbiI6Imh0dHBzOi8vcmFpbi5va3RhMS5jb20iLCJ0eXBlIjoid2ViYXV0aG4uY3JlYXRlIn0=",
-  "stateToken": "00eacMXqkf2pG8K3sBbWqTJNStZpEi9-1Bfwl_mfQT"
```
Comment (mpeng-okta) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md:2619]: Remove `"stateToken"` here as well.

**Review by keithchin-okta — APPROVED** [review]
LGTM, thanks for fixing these! 🚢

**Review by mraible — APPROVED** [review]

**Pull request merged**